### PR TITLE
Video-17-Implement-Remove-From-Cart-Action

### PR DESCRIPTION
**Pull request by basir**

### CODE REVIEW

**Review by yogesh9987 — COMMENTED** [review]
dd

File: frontend/src/reducers/cartReducers.js [hunk 1]
```diff
@@ -15,6 +15,11 @@ export const cartReducer = (state = { cartItems: [] }, action) => {
       } else {
         return { ...state, cartItems: [...state.cartItems, item] };
       }
+    case CART_REMOVE_ITEM:
+      return {
+        ...state,
+        cartItems: state.cartItems.filter((x) => x.product !== action.payload),
```
Comment (ThankLam2411) [inline frontend/src/reducers/cartReducers.js:21]: I don't know why not have result although I tried many ways to run this code

**Review by yogesh9987 — COMMENTED** [review]
index.tsx:25 
        
       No routes matched location "/cart"

**Review by yogesh9987 — COMMENTED** [review]
44

**Pull request merged**